### PR TITLE
feat(package-page): update report links

### LINKED ISSUE #737: We should add "Report Abuse" link to the package page
"Report library" should redirect to https://github.com/cdklabs/construct-hub/issues/new.
Tooltip: "Open an issue in the Construct Hub  repository to report a library"

"Report Abuse" should redirect to abuse@amazonaws.com
Tooltip: "For more information see https://aws.amazon.com/terms/" 

Use cases:
* Report library - a publisher wants to us to remove their deprecated library
* Report abuse - a consumer found a an abusive library
* Provide feedback - a consumer wants to provide feedback to the library's publisher

The UX for this task is not finalized yet

### PR DESCRIPTION
**Pull request by Chriscbr**
Fixes #737 

Screenshots:
Screen Shot 2021-11-23 at 1 29 24 PM
Screen Shot 2021-11-23 at 1 48 17 PM

### CODE REVIEW

File: src/components/ExternalLink/ExternalLink.tsx [hunk 1]
```diff
@@ -15,6 +16,10 @@ export interface ExternalLinkProps extends LinkProps {
    * Adds the nofollow annotation to the anchor's rel attribute
    */
   noFollow?: boolean;
+  /**
+   * Show a custom icon next to the link. `hasIcon` must be false
+   */
+  rightIcon?: ReactNode;
```
Comment (gabewomble) [inline src/components/ExternalLink/ExternalLink.tsx:22]: Why not allow this prop to override hasIcon?

```tsx
const hasIcon = props.hasIcon || Boolean(rightIcon);
const icon =  hasIcon ? rightIcon ?? <ExternalLinkIcon /> : null;

...

{children} {icon}
```

Comment (Chriscbr) [reply src/components/ExternalLink/ExternalLink.tsx]: Whoops, yeah that makes way more sense 👍 Updated.

File: src/views/Package/FeedbackLinks.tsx [hunk 2]
```diff
@@ -1,78 +1,104 @@
...
 };
 
 export const FeedbackLinks: FunctionComponent = () => {
   const state = usePackageState();
+
+  // divider orientation doesn't currently support responsive values
+  const divider = useBreakpointValue({
+    base: <Divider borderColor="white" mr={6} orientation="horizontal" />,
+    md: <Divider borderColor="white" mr={6} orientation="vertical" />,
+  });
```
Comment (gabewomble) [inline src/views/Package/FeedbackLinks.tsx:23]:
```suggestion
const orientation = useBreakpointValue<"vertical" | "horizontal">({
  base: "horizontal",
  md: "vertical"
});

const divider = <Divider borderColor="white" mr={6} orientation={orientation} />
```

File: src/views/Package/FeedbackLinks.tsx [hunk 3]
```diff
@@ -1,78 +1,104 @@
...
-      </Button>
-    );
   }
 
+  // const orientation = { base: "horizontal", md: "vertical" };
+
   return (
     <Stack
       align="center"
+      backgroundColor="#1F50A1"
```
Comment (gabewomble) [inline src/views/Package/FeedbackLinks.tsx:49]: Do we have a value from theme we can use?

**Review by gabewomble — APPROVED** [review]
Two small comments, otherwise LGTM!

File: src/views/Package/FeedbackLinks.tsx (hunk 2 above)
Comment (Chriscbr) [reply src/views/Package/FeedbackLinks.tsx]: Good catch.

File: src/views/Package/FeedbackLinks.tsx [hunk 4]
```diff
@@ -1,78 +1,106 @@
...
-        colorScheme="blue"
+      {repoUrl && (
+        <>
+          <ExternalLink
+            color="currentcolor"
+            data-testid={testIds.githubLink}
+            fontSize="lg"
+            fontWeight="semibold"
+            hasIcon={false}
+            href={`${repoUrl}/issues`}
```
Comment (addihorowitz) [inline src/views/Package/FeedbackLinks.tsx:70]: I believe the link should be ${repoUrl}/issues/new?

File: src/views/Package/FeedbackLinks.tsx [hunk 5]
```diff
@@ -1,78 +1,106 @@
...
+          {divider}
+        </>
+      )}
+      <ExternalLink
+        color="currentcolor"
         data-testid={testIds.reportLink}
+        fontSize="lg"
+        fontWeight="semibold"
+        hasIcon={false}
+        href={`${CONSTRUCT_HUB_REPO_URL}/issues/new`}
```
Comment (eladb) [inline src/views/Package/FeedbackLinks.tsx:85]: A nice addition could have been to inject a title to the new issue with name and version of the package and perhaps some keyword that will make it easier for us to filter.

**Pull request merged**